src/vs/workbench/contrib/issue/browser/issue.ts

```diff
@@ -240,26 +240,6 @@ export class BaseIssueReporterService extends Disposable {
 	}
 
 	public setEventHandlers(): void {
-		this.addEventListener('issue-type', 'change', (event: Event) => {
-			const issueType = parseInt((<HTMLInputElement>event.target).value);
-			this.issueReporterModel.update({ issueType: issueType });
-			if (issueType === IssueType.PerformanceIssue && !this.receivedPerformanceInfo) {
-				this.issueMainService.$getPerformanceInfo().then(info => {
-					this.updatePerformanceInfo(info as Partial<IssueReporterData>);
-				});
-			}
```

Comment:
I'm removing it since it doesn't exist in the issueMainService anymore, but I'm exploring how we can potentially still do this, just unsure if web can support this or not yet. (at least it can't use the `processService` since many there are electron specific)

Reply:
If I'm reading this right... this code runs in the web... but was there an `IIssueMainService` in the web? 🤔 

.
.
.
.
.ah I see: https://github.com/microsoft/vscode/blob/ed62b78db46f84ff29f665957fa1b67675089af1/src/vs/workbench/contrib/issue/browser/issueFormService.ts#L148

it would just throw. Yeah makes sense.